### ISSUE TITLE
CPU or GPU？

### ISSUE THREAD

**Issue by Ethanzhiong** [hustaia/Decoupled_Two-stage_Counting#4]
Does this work require more CPU? The GPU storage utilization rate is not high when conducting model training, but the CPU will be full. Is it because the PMR preprocessing consumes resources?

**Comment by hustaia** [comment]
Yes, the generation of synthesized dot maps consumes lots of CPU resources. To download the pre-generated dot maps, see this link: #3

**Comment by Ethanzhiong** [comment]
thank you！

**Comment by Ethanzhiong** [comment]
Can you give the results after processing the QNRF dataset?

As you said, it will take several days to generate the h5 file

Thank you very much

**Comment by hustaia** [comment]
Train part1
链接：https://pan.baidu.com/s/1sFR_-O-i2SQPueFM1E8ntw 
提取码：2jgl
Train part2
链接：https://pan.baidu.com/s/120M9OAyFUox-cX6KMbw5lA 
提取码：gbnp
Test set
链接：https://pan.baidu.com/s/1Pju0zWAJCNXFVjk0W2Coyw 
提取码：0vpx

**Comment by Ethanzhiong** [comment]
Really appreciate your help!
